snap/snapcraft.yaml

```diff
@@ -117,12 +117,10 @@ parts:
         - pkg-config
 
   chisel:
-    plugin: go
-    source: https://github.com/canonical/chisel.git
-    source-commit: bd27f8700cd7d2a6b4e0df6b10c3761c83a70485
-    build-environment:
-      - CGO: "0"
-      - CGO_ENABLED: "0"
-      - GOBIN: "${CRAFT_PART_INSTALL}/libexec/rockcraft"
-    build-snaps:
-      - go/1.18/stable
+    plugin: nil
+    stage-snaps:
+      - chisel/latest/candidate
```

Comment:
isn't `latest/candidate` a moving target? Might be safer to rollback to the previous code and just use go 1.20 for the skopeo snap too

Reply:
in this case, it isn't safer. We'll need to rethink how we use Chisel.

We also do this for pebble (installing from `latest/stable`).

Bottom-line, in a couple of weeks there will be a new chisel version that will make all Rockcraft release <1.1.0 incompatible with Chisel :s

yes, I know, and there's nothing we can do.

So for now, and until we figure out a way of making Rockcraft critically independent from its dependencies, it is simply easier to always install the latest releases. We have tests for it anyway, so it's actually safer than what we do with pebble.

Reply:
Given the way Chisel works, no. That's actually what we want. I've actually been thinking about getting rid of versions for chisel, and opting instead for a `latest` release only. It's too radical, but that's the safest behavior.

Your concern is valid, but its consequences would be immediately reflected by either a build or tests failure.